clinica/iotools/converters/ukb_to_bids/ukb_utils.py

```diff
@@ -0,0 +1,390 @@
+from os import PathLike
+from typing import Iterable, List, Tuple
+
+from pandas import DataFrame
+
+
+def find_clinical_data(
+    clinical_data_directory: PathLike,
+) -> DataFrame:
+    from pathlib import Path
+
+    import pandas as pd
+
+    from clinica.utils.stream import cprint
+
+    # Read the xls
+    try:
+        image_data_file = list(Path(clinical_data_directory).glob("*.tsv"))
+    except StopIteration:
+        raise FileNotFoundError("Clinical data file not found.")
+
+    # Assign the dfs
+    if len(image_data_file) == 1:
+        df_clinical = pd.read_csv(str(image_data_file[0]), sep="\t")
+        cprint(msg="All clinical data have been found", lvl="info")
+    elif len(image_data_file) == 0:
+        raise FileNotFoundError("Clinical data not found or incomplete. Aborting")
+    elif len(image_data_file) > 1:
+        raise FileNotFoundError("Too many data files found, expected one. Aborting.")
+
+    columns_to_have = [
+        "age_when_attended_assessment_centre_f21003_2_0",
+        "age_when_attended_assessment_centre_f21003_3_0",
+        "year_of_birth_f34_0_0",
+        "age_at_recruitment_f21022_0_0",
+        "age_when_attended_assessment_centre_f21003_2_0",
+        "sex_f31_0_0",
+    ]
+    for column_name in columns_to_have:
+        if column_name not in df_clinical:
+            raise FileNotFoundError(f"column {column_name} not found. Aborting")
+    return df_clinical
+
+
+def read_imaging_data(imaging_data_directory: PathLike) -> DataFrame:
+    from pathlib import Path
+
+    import pandas as pd
+
+    source_path_series_nifti = pd.Series(
+        find_imaging_data(imaging_data_directory), name="source_path"
+    )
+    source_path_series_dicom = pd.Series(
+        find_dicom_data(imaging_data_directory), name="source_path"
+    )
+    # justifications regarding the chosen files can be found in clinica's documentation: https://github.com/aramis-lab/clinica/blob/dev/docs/Converters/UKBtoBIDS.md
```

Comment:
Silence PEP8 + rewording:

```suggestion
    # Rationales for this choice of files can be found in the documentation:
    # https://github.com/aramis-lab/clinica/blob/dev/docs/Converters/UKBtoBIDS.md
```